python/taichi/main.py

```diff
@@ -117,19 +129,11 @@ def main(debug=False):
     with open(name) as script:
       script = script.read()
     exec(script, {'__name__': '__main__'})
-  elif mode == "test_python":
-    return test_python(test_files=sys.argv[2:])
-  elif mode == "test_cpp":
-    return test_cpp(test_files=sys.argv[2:])
   elif mode == "test":
-    if test_python(test_files=sys.argv[2:]) != 0:
-      return -1
-    if len(sys.argv) <= 2:
-      return test_cpp()
-  elif mode == "test_verbose":
-    if test_python(test_files=sys.argv[2:], verbose=True) != 0:
-      return -1
-    return test_cpp()
+    ret = test_python(test_files=args.files, verbose=args.verbose)
+    if ret: exit(-1)
+    ret = test_cpp(test_files=args.files)
+    return ret
```

Comment:
Here comes the problem: How can we distinct cpp test_files and python test_files? Still need test_cpp cmd?

Reply:
Hmm... Maybe use `test -py [files]` and `test -cpp [files...]` to specify the files? It will be much cleaner if a single root command `test` can be used.